resource-managers/kubernetes/lightbend-build/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.12
```

Comment:
Let's bump this to `1.1.X`. We need to convert all projects to use SBT version > 1.0, because of changes in the way you query settings, which I've needed to invoke in the release build.

Reply:
Will fix.